src/parsec_service/install_parsec_linux.md

```diff
@@ -15,48 +15,62 @@ can be a space or comma-separated subset of: `mbed-crypto-provider`, `pkcs11-pro
 `tpm-provider`. Choose the providers you want to install depending on what is available on the
 platform.
 
-Create and log in to a new user named `parsec`.
+Create the Parsec socket directory.
 
 ```
-sudo useradd -m parsec
-sudo passwd parsec
-su --login parsec
+mkdir /tmp/parsec
 ```
 
-In its home directory, pull and install Parsec as a daemon.
+In a deployment **without an Identity Provider**, create the `parsec-clients` group and set the
+correct permissions on the socket folder. Mutually trusted Parsec Clients will need to be in that
+group.
 
 ```
-git clone https://github.com/parallaxsecond/parsec.git
-cargo install --features $DESIRED_FEATURES --path parsec
+sudo groupadd parsec-clients
+sudo chown :parsec-clients /tmp/parsec
+sudo chmod 750 /tmp/parsec
 ```
 
-Copy and adapt the [configuration](configuration.md) you want to use. For a secure deployment, make
-sure to activate the `log_error_details` option and to use a `trace` log level.
+For example, adding the imaginary `parsec-client-1` user to the `parsec-clients` group:
 
 ```
-cp parsec/config.toml config.toml
+sudo usermod -a -G parsec-clients parsec-client-1
 ```
 
-Create the Parsec socket directory.
+Users just added to that group might need to log-out and log-in again to make sure the change apply.
```

Comment:
I think we need to shout a bit louder about this login/logout cycle, because that would be easy to miss by someone who is following this recipe and mainly just looking for the commands that they need to type. Blurb in between the commands would be easy to glance over without really thinking about it. Perhaps flag it with a bold "Important", or better still add a paragraph right at the end (after referring the e2e_tests) with a cautionary note along the lines of: if you see PermissionDenied errors from the client, double check that the group change has taken effect, which you can check by calling the `groups` command with no arguments.

Reply:
Will change!

Reply:
> with a cautionary note along the lines of: if you see PermissionDenied errors from the client, double check that the group change has taken effect

On as similar note, maybe we should also create a troubleshooting page - "if you see _this_ error, do _this_" or "you haven't done _that_"